semgrep/semgrep/core_runner.py

```diff
@@ -635,7 +653,7 @@ def _run_rules_direct_to_semgrep_core(
                 )
         # end for rule, language ...
 
-        return outputs, {}, errors, set(Path(p) for p in target_manager.targets), {}
+        return outputs, {}, errors, all_targets, match_time_matrix
```

Comment:
It looks like the result here has changed. Is `set(Path(p) for p in target_manager.targets)` and `all_targets.union(targets)` the same thing?

Reply:
set(Path(p) for p in target_manager.targets) was incorrect because the Set here only contained the directory we were running. I believe all_targets is the list of targets actually run against